internal/auth/internal/types/traffic_permissions_test.go

```diff
@@ -441,6 +441,82 @@ func permissionsTestCases() map[string]permissionTestCase {
 			},
 			expectErr: `invalid element at index 0 of list "destination_rule": prefix values, regex values, and explicit names must not combined`,
 		},
+		"destination-rule-empty": {
```

Comment:
Do we have any happy path tests for these?

Reply:
added